plugins/tours/tests/phpunit/integration/renderTourComments.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Tests for render_tour_comments().
+ *
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @package     spiralWebDb\CornerstoneTours\Tests\Integration
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Integration;
+
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+use function spiralWebDb\CornerstoneTours\render_tour_comments;
+
+/**
+ * @covers ::\spiralWebDb\CornerstoneTours\render_tour_comments
+ *
+ * @group   tours
+ */
+class Tests_RenderTourComments extends Test_Case {
+
+	/**
+	 * @dataProvider addTestData
+	 */
+	public function test_should_echo_meta_key_values_when_postmeta_exists( $post_data, $meta ) {
+		// Create and get the $tour_id using WordPress' factory method.
+		$tour_id = $this->factory->post->create( $post_data );
+
+		// Add post_meta to the database so we can call it.
+		add_post_meta( $tour_id, 'tour_comments', $meta['tour_comments']
+		);
```

Comment:
Instead of hard coding the meta, allow for flexibility by registering the meta that's passed from the test data:

```php
foreach( $meta as $key => $value ) {
	add_post_meta( $tour_id, $key, $value );
}
```

This approach gives you flexibility to add more test data edge cases and scenarios without having to modify the test method's code.